rollback_on_fail.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+rollback () {
+    curl -L -o - https://github.com/Jellyfish-AI/jf_agent/archive/refs/tags/stable.tar.gz | tar xz --directory ./
+    rm -r jf_agent/
+    mv jf_agent-stable/jf_agent/ jf_agent/
+    python jf_agent/main.py "$@"
+}
+
+echo "checking for time limit override"
+if [[ -v OVERRIDE_TIME_LIMIT ]]; then
+	echo "Time limit override detected, agent will timeout at ${OVERRIDE_TIME_LIMIT}"
+	TIME_LIMIT="${OVERRIDE_TIME_LIMIT}"
+else
+	TIME_LIMIT=$(curl -H "Accept: application/json" -H "Content-Type: application/json" -H "Jellyfish-API-Token: $JELLYFISH_API_TOKEN" https://app.jellyfish.co/endpoints/agent/time-limit | jq -r .time_limit_string)
+	TIME_LIMIT="${TIME_LIMIT:-12h}"
+	echo "Time limit override not detected, retrieved timelimit from Jellyfish: (${TIME_LIMIT})"
+fi
+echo "running agent with timelimit $TIME_LIMIT"
+
+timeout --preserve-status "$TIME_LIMIT" python jf_agent/main.py "$@" 
+CODE=$?
+
+if [[ $CODE -ne 0 && -z "${ROLLBACK_OVERRIDE}" ]]; then
```

Comment:
```suggestion
if [[ $CODE -ne 0 && -v ROLLBACK_OVERRIDE ]]; then
```
Same trick as above, use `-v` when assessing "is this variable set?"